doc/source/guide/spatial-partitioning.ipynb

```diff
@@ -0,0 +1,717 @@
N5hE1eU8TwAFTBtCWWDHDLVUqCKMIsNwdnz7wrG/9ltNf/dC0n8/bKo4fG99/3a1zTz6S7D4H4FObuzLDNqfappyFjnvVr3N2Nyhq9zgNYF83DsSgS904jmFjSLKGJhxEA9Zv1GrjCkNXo0wqhdFEEB4bySHlabHCIQ5ipp4qFxni/CKLXSWm01niNQESJSAuKGvfZWUdrJ8QZQDxWA7hzgmIlCsjxqqELRFr1YVByJOnrXm7gRBNp+zHxlML5XNLbhiqlpZnAohKkuusLhvPCVkEX7J2n83ZdKH+WJ+arbx5f8p+T8YS4eGce+buuco3AagVgT+Uuv2IseYwrMuznvOGz3zqxIXpSHHbwsgaOhgxvCz898VUcGG8EMQoBBGiTbjGd3OEmoN/sYjU7m4d71ZxTp5RO3CRJzvNzXllw4yrnjqDiblFPHrLDahkOsvJVUCAilSj41m5MJO3e5YgLisU9tkSE0KhowyArCIAm0k0BgAIlIVAWigCWYGVuL1Ozf/dsDJzCcWQkcsr2r3T4pPV5cwkpRR+HFv+TGSPPRXZzyhIa6oq/CpeW+evudabjmpulWKyi9GILUiiZIcgGaWspk5v6z1Dq+0vyBJwLcDF2jmAq8DhU5eXFs8u+uPrXnjCuGtnGaItAwCXuBgy5QhQEcfl075cTuBX4NRzL6iR83utwtFREcxUULnxFT//l4c//Rs/fKydYw8LA3GLqyhOffhyCd8yncHVPU7YxnUPs6rECGfLcHf01bNgwBCKbhpMhGy05azVtwTDaAUMdC5Arnec5F+2NphtnLHaKpZWr6paJHB1G1/yiCOoYhHHeiNCrDgVM3vDOx2zFqmT23T8O4gwgp0vNl3PNTNZNjh0QP5CvpACAEfJgiJhPzR19U2T/tLcU6N7bnTUsjsAQ4eBiBod12BIqFqNuwBiDHEs7T5hRn7bBEnk2qzKMYlM6637RyYOLpRtr+PwAmXfR9n3cXFWh/AeszAzlctF4yNZdlKZwPNc4XmesCyrozpcZloWkpL0LKr24xAH1W0elc6zdwp5aq+QZ0RiMjGjxI4Lyr4mqotxyATEYyMIdkxAec7y8ZuWw0mtNWHsEQHRRGr/WDE+X4Dy149wkSTAaWtCoTaMFjMoa9ElX/GYZKQsQpi1xOUvLvjPvinnPvpUObpqPpJ7anb/Uur2Iyc3dEGGK4oTl5f2E1Hcz+gFtiUwkXExkXHhRxKFIEIxjKGG2HCqUwQxDtIl7KA8nlR7UNy87ndFMbJUwLO+8gCevP4QLuzd3bYhHgPIs4/pjM0z+e53QceEwl47xpjofPJCEGI/KG9oPFyLrajiBc5SXHEsJViMS1sKYhCYBIEEMUAM1v+TAEDEJLTt+/KNlIxUXGtEwaQjMzBJZkjFpEpK2BUpxBlpHXSkUPsi69Nu6FyOYytcFDz1lK1eXSa+YTPXE8DCLHsAPDgskaIIKcRoLghvHAa5h3aOuSnHKj0+U1pX2HO0e4QdtV+MZW3iYEqULgUxBUxpALAhZ8pwD86r9AKDrAJ714+K4DoARgRej0PvvFMA+GzMeO6/z5Stl0+lcXOud4YKjdxTZCFE7Fiwx4fZGKTbEEpOGgxCLtpsNktDPf1KmNYJhO5ZvipAbPb6LEDWxn+peSqrLlyrjq8AESjOCrAkEBR402XYSjjzS63CUCy7+k2SurwERAtL+b2jUfnksy49etfn9j39rYHljuUy5QtjQfHS7vLCuWTXew8fP3am91dg2OJUJw2eAPA4gJsGWJZhoGs92AZGFYY+o6j3rqWdQmDuRtI65VfKT0ocRN0kom1Z8FwXKc9FyvVq/nfheS4ce2VIkDhuxgyIpP1HYoy0POEd8Uq8YQWIi8q6ek5Zex1iXzKcALRKZGciROMjCHeMQ7nt3n5Cxe6vc4AieN6eHMKZMmSxeXJliyj6/9n78yDJruvME/zOvW/zPfaIjFwiE8jESgDcmRJFslRSSaJSUjWqqpep6rYe1thUz0zPTLXhjza0tbUZu9rGLG16qaopm5q2sipJJZU2ihQoiamFOwkIBEkQBEAQewKRe+wRvr/t3jN/PPdIj0iPCI/w50tEvJ8hEBnuz++7vr1773fP+U6nlwXmu6/bCcd40yAE79WCiwaR+3DW/O6Sr05frQUX1wM9UzDF0lzKaBaKS7J2Ejrmgam8FIK++e5K+T/uywm3OkXAMSUcU2KCgaofRQfXOixsfBhIw8UHxDwWeBQ3eApqOON/hhKhFM6/eRVjK+t4+6HzCOzOtJ8NdnHCTEvDYIQdlR7dmzEZib9ZOnjkutSqslytdR2dLH3Dr7vWNABcDUwsq/0s26OBRRAjBbrugmc0k6EAESUpR6fY4cHnbgD/pS1wC5Yij/hUHAHH5ZaVegCJgCXKAKwWQVjELAifGsvAMAy8vVDc0fo/IwV4B/vI3TCI6znJy2cc4b9T55kMvFdGhPv6LV34R4pF2iS1mkZwwyC92u3z6DeDiOxJI1pc3mLgzDdX66grjY8U+ivIBmt1kCEgeyhADyM10wEIyPqJEBwnDUsItV//114ho4JwFFMGSWwc5LK/k+fvYUIDYjff4O2Q1rDWSrsf07JLGYBMrZXOLsw/9/Hbrz1bsjLZJ5bf/dL9pdvztgoCAPCFYQBYAvDtgz+ThGNEq/qRiMAJRwpDq5ovjIOnTMdIKvQWJWu/YqYOWrW6I0KlENbrqNbbZ4QJIeDcFYnZtkzDsRtisWXB6kC4DQA7YNqi2jIRgtF8JP6a7ZcdOy3ta4YNTf2dxiiGaRmigqlMNjAFwvX2ieoC6DSbhomguFHkzhZwSyGPzjrGq+fT5luTllzPumH502Opb01YsgwAnmYDjawd57OPJVk7CZ2SPVlIvffuSnmgnSACsraBrG1AaUbFC1D2Qniq68DJIYAxQ2sYozLmeQZrfE+Ny4RdGFtZw0d+8GO8/dB5rE2O73l8ES4Uw8inw/payTxwCDYBmJAKs4ZCqgv
```

Comment:
It's not super clear to me what is meant with this advice. Do you mean a workflow (eg a groupby) that might increase the number of partitions along the workflow?

Reply:
I'm not sure which part of the notebook you are referring to here?

Reply:
Whoops, no idea how that got commented so out of place ..

Reply:
Actually, so if you look in diff in the "Files changed" tab, then it appears at the correct location ..

Reply:
gotcha, the diff didn't load previously because of the notebook size.
I see what you mean about point number 3 - it's not very clear.
Yes, changing the number of partitions or even reverting back to GeoPandas could be useful after running an operation such as a  groupby.
I'll update the notebook later today.